scripts/spack/packages/axom/package.py

```diff
@@ -212,6 +227,13 @@ def initconfig_hardware_entries(self):
                     cudaflags += " -std=c++14"
                 else:
                     cudaflags += " -std=c++11"
+
+                # SERAC EDIT BEGIN - debug flag causes the NVIDIA assembler to fail
+                # NVCC ignores the host compiler when linking??
+                # and some MPI -Wl,-rpath, flags are added from somewhere
+                cudaflags += " -ccbin ${CMAKE_CXX_COMPILER} -forward-unknown-to-host-compiler "
```

Comment:
I'm concerned why this is now showing up when it never has before.  What was led to this fix?

Reply:
So this was not showing up for us before because I had punted (in #254) on separable compilation, which axom+raja+cuda requires (not req'd by axom+cuda~raja).  I think it didn't show up for other Axom users because we're the only ones who use axom+cuda but don't build our project with NVCC/as CUDA C++ (which necessitated LLNL/axom#494).

Reply:
I thought cmake added the ccbin option automatically due to setting `CMAKE_CUDA_HOST_COMPILER` which we do through BLT here:

https://github.com/LLNL/blt/blob/68c98b7306e0b1f19cb383ac6af210d6f75627fc/cmake/thirdparty/SetupCUDA.cmake#L33